lib/syncthing/utils.go

```diff
@@ -168,11 +168,15 @@ func OpenDatabase(path string) (newdb.DB, error) {
 	return sdb, nil
 }
 
-func TryMigrateDatabase(sdb newdb.DB, miscDB *db.Typed, oldDBDir string) error {
+// Attempts migration of the old (LevelDB-based) database type to the new (SQLite-based) type
+func TryMigrateDatabase(sdb newdb.DB, oldDBDir string) error {
+	miscDB := db.NewMiscDB(sdb)
```

Comment:
Move down to where it's actually needed?

Reply:
Sure..